### PR TITLE
Auto on session tracking

### PR DESCRIPTION
**Pull request by Cawllec**
## Goal

Update creation and configuration of Bugsnag client to match changes made [here](https://github.com/bugsnag/bugsnag-php/pull/517).

Note: composer.json changes are temporary to aid testing, and must be removed before merging.

Requires: https://github.com/bugsnag/bugsnag-php/pull/517

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -10,12 +10,22 @@
             "email": "notifiers@bugsnag.com"
         }
     ],
+    "repositories": [
```
Comment (GrahamCampbell) [inline composer.json]: Why are these added? If you want to checkout a branch, just do `"bugsnag/bugsnag": "dev-base-changes",`. No need for extra repos bypassing packagist.

Comment (Cawllec) [reply composer.json]: Trying without specifying the repositories causes an issue, as it can't seem to find the branch.
